pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -228,6 +230,31 @@ func getProfileVersion(config *checkconfig.CheckConfig) uint64 {
 	return profileVersion
 }
 
+func getDeviceType(store *metadata.Store) string {
+	deviceType := strings.ToLower(store.GetScalarAsString("device.type"))
+	switch deviceType {
+	case
+		"access_point",
+		"firewall",
+		"load_balancer",
+		"pdu",
+		"printer",
+		"router",
+		"sd-wan",
+		"sensor",
+		"server",
+		"storage",
+		"switch",
+		"ups",
+		"wlc":
+		return deviceType
+	case "":
+		return "other"
+	}
+	log.Debugf("Unsupported device type: %s", deviceType)
+	return "other"
```

Comment:
I still think there is no much value of having "other" as default :) 

Why not just keep empty string as default value?

Trying to better understand what are the benefits of have "other" as default value.